test/t8_geometry/t8_test_point_inside.cxx

```diff
@@ -243,12 +290,71 @@ t8_test_point_inside_specific_triangle ()
   t8_forest_unref (&forest);
 }
 
+/* In this test we define a quad in the x-y plane
+ * and a point that lies in a quad that is parallel
+ * to this triangle on the z-axis.
```

Comment:
Which triangle? We are talking about two quads

Reply:
fixed